msdk-rawdata/src/test/java/io/github/msdk/filtering/cropper/CropFilterMethodTest.java

```diff
@@ -0,0 +1,63 @@
+/* 
+ * (C) Copyright 2015 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package io.github.msdk.filtering.cropper;
+
+import com.google.common.collect.Range;
+import io.github.msdk.MSDKException;
+import io.github.msdk.datamodel.datapointstore.DataPointStore;
+import io.github.msdk.datamodel.datapointstore.DataPointStoreFactory;
+import io.github.msdk.datamodel.rawdata.MsScan;
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+import io.github.msdk.io.rawdataimport.mzml.MzMLFileImportMethod;
+import java.io.File;
+import java.util.List;
+import org.junit.Test;
+import org.junit.Assert;
+
+public class CropFilterMethodTest {
+
+    private static final String TEST_DATA_PATH = "src/test/resources/";
+
+    @SuppressWarnings("null")
+    @Test
+    public void testCropFilter() throws MSDKException {
```

Comment:
It's always useful to have multiple tests, including some borderline cases (e.g., what happens if you specify a range that no scan falls into?)